lib/util.js

```diff
@@ -478,9 +478,23 @@ function formatValue(ctx, value, recurseTimes) {
   if (ctx.showHidden) {
     keys = Object.getOwnPropertyNames(value);
   } else {
-    keys = Object.keys(value);
-    if (symbols.length !== 0)
-      symbols = symbols.filter((key) => propertyIsEnumerable.call(value, key));
+    // this might throw if `value` is a Module Namespace Object from an
+    // unevaluated module, but we don't want to perform the actual type
+    // check because it's expensive.
+    try {
+      keys = Object.keys(value);
+    } catch (err) {
+      if (err instanceof ReferenceError) {
+        keys = Object.getOwnPropertyNames(value);
```

Comment:
Why does this not throw compared to Object.keys? Because of https://www.ecma-international.org/ecma-262/6.0/#sec-enumerableownnames 5.a.i.?

Reply:
[[GetOwnProperty]] -> [[Get]] -> [[GetBindingValue]] which throws

Reply:
☝️ the `Object.getOwnPropertyNames(value)` fallback should filter non-enumerable to be equiv to `keys`